src/tests/topp/FileFilter_18_output.consensusXML

```diff
@@ -55,12 +53,6 @@
 	<IdentificationRun id="PI_2" date="2011-02-02T13:06:01" search_engine="Mascot" search_engine_version="2.2.04">
 		<SearchParameters db="regulatedPeptides" db_version="regulatedPeptides_appended_20100831.fasta" taxonomy="" mass_type="monoisotopic" charges="" enzyme="trypsin" missed_cleavages="2" precursor_peak_tolerance="10" precursor_peak_tolerance_ppm="false" peak_mass_tolerance="0.5" peak_mass_tolerance_ppm="false" >
 			<FixedModification name="Carbamidomethyl (C)" />
-			<FixedModification name="Carbamidomethyl (C)" />
-			<FixedModification name="Label:13C(6)15N(2) (K)" />
-			<FixedModification name="Label:13C(6)15N(4) (R)" />
```

Comment:
The others were duplication only, but here a new ones are added! see https://github.com/OpenMS/OpenMS/blob/develop/src/tests/topp/FileFilter_18_input.consensusXML#L50